bin/docker-setup.sh

```diff
@@ -114,26 +110,29 @@ cli wp plugin install wordpress-importer --activate
 echo "Importing some sample data..."
 cli wp import wp-content/plugins/woocommerce/sample-data/sample_products.xml --authors=skip
 
-echo "Activating the WooCommerce Payments plugin..."
+echo "Activating the WooPayments plugin..."
 cli wp plugin activate woocommerce-payments
 
-echo "Setting up WooCommerce Payments..."
+echo "Setting up WooPayments..."
 if [[ "0" == "$(cli wp option list --search=woocommerce_woocommerce_payments_settings --format=count)" ]]; then
-	echo "Creating WooCommerce Payments settings"
+	echo "Creating WooPayments settings"
 	cli wp option add woocommerce_woocommerce_payments_settings --format=json '{"enabled":"yes"}'
 else
-	echo "Updating WooCommerce Payments settings"
+	echo "Updating WooPayments settings"
 	cli wp option update woocommerce_woocommerce_payments_settings --format=json '{"enabled":"yes"}'
 fi
 
+echo "Installing and activating Disable WordPress Updates..."
+cli wp plugin install disable-wordpress-updates --activate
+
```

Comment:
Any specific reason to include this plugin and disable WP updates? If you're setting up the containers for the first time or re-creating them, new WP versions are installed automatically.

Reply:
I have this plugin active at all times (and probably most of the devs in Moltres) to avoid having my cloned WooPayments overwritten by automatic updates if the current checked-out version is older than the latest release. 

Also, we should be self-conscious when managing the various versions we are using in our local dev for WP, Woo, etc. I, for one, disable that plugin, update what I want, and activate it again.

Third, not having the updates checks provides a modest performance boost.

I am not strongly for including this, but it does help prevent our local environment from being broken by automatic updates.

Reply:
Sure. Since there is a valid use case, we can keep it :)